src/ImageService.Models.Db/DbImagesProject.cs

```diff
@@ -17,7 +17,7 @@ public class DbImagesProject
 		public Guid CreatedBy { get; set; }
 	}
 
-    public class DbImageConfigurationProject : IEntityTypeConfiguration<DbImagesProject>
+    public class DbImagesConfigurationProject : IEntityTypeConfiguration<DbImagesProject>
```

Comment:
why did you renamed it?

Reply:
will change names in other task to avoid conflicts

Reply:
typo

Reply:
configuration have to be last word